### ISSUE TITLE
Send the output of the FFE into a single-pass MLSD checker

### ISSUE THREAD

**Issue by sgherbst** [StanfordVLSI/dragonphy#43]
Depends on issue #42.  As with the FFE:
1. Need a path to configure the MLSD externally (e.g., setting thresholds and coffecients)
2. Need to be able the bypass MLSD when sending data to the SRAM, PRBS checker, etc.
I would also suggest that we have registers for monitoring the activation rate of this feature (start monitoring, stop monitoring, monitor duration, etc.)

**Comment by zamyers** [comment]
1)  was already done in the flat_forward_path.sv. However, it looks like this file got clobbered somewhere and an old version (not working version) sits on the master.

**Comment by sgherbst** [comment]
Sorry, to clarify I meant integrating this feature into the ``dragonphy_top`` module.  That does depend on #32 but hopefully I will have finished that today.
